include/geode/geometry/point.h

```diff
@@ -236,8 +236,30 @@ namespace geode
             return dimension;
         }
     };
+
+    template < index_t dimension >
+    class OpenGeodePointException : public OpenGeodeException
+    {
+    public:
+        template < typename... Args >
+        explicit OpenGeodePointException(
+            Point< dimension > point_in, const Args &...message )
+            : OpenGeodeException{ absl::StrCat( message... ) },
+              point{ std::move( point_in ) }
```

Comment:
Does the point coordinates be shown in the message by default ?

Reply:
Yes, see the last version